jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/log/RuntimeLogPresenter.java

```diff
@@ -53,64 +67,119 @@ public IsWidget getWidget() {
+                                           logType))
+                    .filter(Objects::nonNull)
+                    .collect(toList());
+
+            view.setLogs(logsLine);
+        }).getProcessInstanceLogs(serverTemplateId,
+                                  deploymentId,
+                                  processInstanceId);
+    }
 
+    protected String getLogline(RuntimeLogSummary logSummary,
+                                LogType logType) {
         if (LogType.TECHNICAL.equals(logType)) {
-            processRuntimeDataService.call(new RemoteCallback<List<RuntimeLogSummary>>() {
-                @Override
-                public void callback(List<RuntimeLogSummary> logs) {
-                    final List<String> logsLine = new ArrayList<String>();
-
-                    if (logOrder == LogOrder.DESC) {
-                        Collections.reverse(logs);
-                    }
-
-                    for (RuntimeLogSummary rls : logs) {
-                        logsLine.add(rls.getTime() + ": " + rls.getLogLine() + " - " + rls.getType());
-                    }
-
-                    view.setLogs(logsLine);
-                }
-            }).getRuntimeLogs(currentServerTemplateId,
-                              currentDeploymentId,
-                              currentProcessInstanceId);
+            String agent = constants.System();
+            if ((NODE_HUMAN_TASK.equals(logSummary.getNodeType()) && logSummary.isCompleted()) ||
+                    (NODE_START.equals(logSummary.getNodeType()) && !logSummary.isCompleted())) {
+                agent = constants.Human();
+            }
+            return LOG_TEMPLATES.getTechLog(
+                    DateUtils.getDateTimeStr(logSummary.getDate()),
+                    logSummary.getNodeType(),
+                    SafeHtmlUtils.fromString(logSummary.getNodeName()),
+                    (logSummary.isCompleted() ? " " + constants.Completed() : ""),
+                    agent).asString();
         } else {
-            processRuntimeDataService.call(new RemoteCallback<List<RuntimeLogSummary>>() {
-                @Override
-                public void callback(List<RuntimeLogSummary> logs) {
-                    final List<String> logsLine = new ArrayList<String>();
-                    if (logOrder == LogOrder.DESC) {
-                        Collections.reverse(logs);
-                    }
-
-                    for (RuntimeLogSummary rls : logs) {
-                        logsLine.add(rls.getTime() + ": " + rls.getLogLine());
-                    }
-
-                    view.setLogs(logsLine);
-                }
-            }).getBusinessLogs(currentServerTemplateId,
-                               currentDeploymentId,
-                               currentProcessName,
-                               currentProcessInstanceId);
+
+            String prettyTime = moment().milliseconds(logSummary.getDate().getTime() - new Date().getTime()).fromNow();
```

Comment:
You probably don't need to do getTime minus current getTime, fromNow should tell you that already...